proposals/16-first-filecoin-storage-deal.md

```diff
@@ -0,0 +1,130 @@
+# Proposal: Reduce friction and time to first finalized deal for new Filecoin storage clients
+
+Authors: @terichadbourne
+
+Initial PR: https://github.com/protocol/web3-dev-team/pull/16 <!-- Reference the PR first proposing this document. Oooh, self-reference! -->
+
+<!--
+This template is for a proposal/brief/pitch for a significant project to be undertaken by a Web3 Dev project team.
+The goal of project proposals is to help us decide which work to take on, which things are more valuable than other things.
+-->
+<!--
+A proposal should contain enough detail for others to understand how this project contributes to our team’s mission of product-market fit
+for our unified stack of protocols, what is included in scope of the project, where to get started if a project team were to take this on,
+and any other information relevant for prioritizing this project against others.
+It does not need to describe the work in much detail. Most technical design and planning would take place after a proposal is adopted.
+Good project scope aims for ~3-5 engineers for 1-3 months (though feel free to suggest larger-scoped projects anyway). 
+Projects do not include regular day-to-day maintenance and improvement work, e.g. on testing, tooling, validation, code clarity, refactors for future capability, etc.
+-->
+<!--
+For ease of discussion in PRs, consider breaking lines after every sentence or long phrase.
+-->
+
+## Purpose &amp; impact 
+#### Background &amp; intent
+As demonstrated in recent walkthroughs by Andrew Nesbitt and Jonathan Victor, there are a number of critical hurdles blocking an easy path to storing files on Filecoin, including both engineering issues (e.g. Powergate doesn't work), UX issues (lack of description in CLI about what stages exist, showing miners who aren't actually accepting deals, etc.), and documentation issues (incorrect or incomplete instructions for steps in the process that otherwise are functional).
+
+Since our goal is to promote use of the full W3DT stack by developers with storage needs (as opposed to miners), and we aim to include Filecoin in that journey, most paths to adoption are blocked until it's possible to quickly and easily make one's first storage deal. 
+
+#### Assumptions &amp; hypotheses
+- It should be possible to store a file on Filecoin
+- Documentation should accurately reflect the recommended actions
+- Developers who attempt to use our product now and fail are unlikely to trust us in the future, once the experience has improved
+
+#### User workflow example
+A developer using Filecoin for the first time would visit docs.filecoin.io and successfully follow instructions there to acquire Filecoin to use for the deal, sync or copy the chain (with an understanding of trade-offs in this decision), view available miners with useful information about pricing and availability for storage deals, negotiate a deal, transfer their file, and receive confirmation that the data has been stored successfully. 
```

Comment:
Good lifecycle description! Agree we need docs that walk through the whole journey step by step